glue/core/roi.py

```diff
@@ -120,6 +126,14 @@ def to_polygon(self):
         as a polygon."""
         raise NotImplementedError()
 
+    def rotate_to(self, theta):
+        """Set rotation angle of ROI around center to theta"""
```

Comment:
We should make it clear if we accept degrees or radians.

Reply:
Ideally quantities would be accepted as well (defaulting to radian if no unit is present), but units support in Glue at this point seems very limited, so I am not sure if it makes sense to introduce it at one single point.